.appveyor.yml

```diff
@@ -53,6 +53,12 @@ after_build:
   - python ../../setup.py EIGEN3_INCLUDE_DIR=C:/projects/eigen build --build-dir=.. --skip-build bdist_wheel install
 
 test_script:
+  # Cmake (C++) unit tests
+  - cmd: set PATH=%BOOST_LIBRARYDIR%;%PATH%
+  - cmd: cd %APPVEYOR_BUILD_FOLDER%\build
+  - cmd: ctest -C %configuration% --output-on-failure --timeout 300
+  # Python unit tests
+  - cmd: cd %APPVEYOR_BUILD_FOLDER%\build\python
```

Comment:
I think this last `cd` isn't really necessary

Reply:
I thought that originally too, but the python testing that runs on the following line seems to need to be run from that directory. Here is the AppVeyor run without the cd command: https://ci.appveyor.com/project/danielh/dynet-c3iuq/build/1.0.67 (commit https://github.com/clab/dynet/pull/1013/commits/2ce679b0b328b8b5e3c8c700f00fb804728a0701). I wasn't sure what causes the errors, I'm not familiar enough with python unit testing.

Reply:
Oh, right, it was picking up the `dynet` directory inside `build` because that was the current working directory. So I guess it is necessary to `cd` to a directory without a `dynet` in it :)

Reply:
Oh, funny. You're right, that must be it.